packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -1305,6 +1311,22 @@ export class Table extends TableBase {
     });
   }
 
+  /**
+   * Get schema attributes of table or index.
+   *
+   * @returns Schema of table or index.
+   */
+  public schema(indexName?: string): SchemaOptions | undefined {
```

Comment:
It's a little weird that this returns `SchemaOptions | undefined`, right? Especially in the case that you don't provide `indexName`, because in that case `SchemaOptions` will always exist.

Should we make it return `SchemaOptions` always, and throw an exception if an index with the given name was not found?

Reply:
Yeah, it will be much better.